docs/guides/using-organizations.md

```diff
@@ -0,0 +1,133 @@
+> non-standard enterprise license to use. Do not use organizations on your
+> production instance!
+>
+> For more details, [contact your account team](https://coder.com/contact).
+
+Organizations allow you to run a Coder deployment with multiple platform teams,
+all with uniquely scoped templates, provisioners, users, groups, and workspaces.
+
+## Prerequisites
+
+- Coder deployment with non-standard license with Organizations enabled
+  ([contact your account team](https://coder.com/contact))
+- User with `Owner` role
+- Coder CLI installed on local machine
+
+## Switch to the preview image and enable the experiment
+
+To try the latest organizations features, switch to a preview image in your Helm
+chart and enable the [experimental flag](../cli/server.md#--experiments).
+
+For example, with Kubernetes, set the following in your `values.yaml`:
+
+```yaml
+coderd:
+  image:
+    repo: ghcr.io/coder/coder-preview
+    tag: orgs-preview-aug-16
+  env:
+    - name: CODER_EXPERIMENTS
+      value: multi-organization
+```
+
+> See all
+> [preview images](https://github.com/coder/coder/pkgs/container/coder-preview)
+> in GitHub. Preview images prefixed with `main-` expire after a week.
+
+Then, upgrade your deployment:
+
+```sh
+helm upgrade coder coder-v2/coder -f values.yaml
+```
+
+## The default organization
+
+All Coder deployments start with one organization called `Default`.
+
+To edit the organization details, navigate to `Deployment -> Organizations` in
+the top bar:
+
+![Organizations Menu](../images/guides/using-organizations/deployment-organizations.png)
+
+From there, you can manage the name, icon, description, users, and groups:
+
+![Organization Settings](../images/guides/using-organizations/default-organization.png)
+
+## Guide: Your first organization
+
+### 1. Create the organization
+
+Within the sidebar, click `New organization` to create an organization. In this
+example, we'll create the `data-platform` org.
+
+![New Organization](../images/guides/using-organizations/new-organization.png)
+
+From there, let's deploy a provisioner and template for this organization.
+
+### 2. Deploy a provisioner
+
+[Provisioners](../admin/provisioners.md) are organization-scoped and are
+responsible for executing Terraform/OpenTofu to provision the infrastructure for
+workspaces and testing templates. Before creating templates, we must deploy at
+least one provisioner:
+
+using Coder CLI, run the following command to create a key that will be used to
+authenticate the provisioner:
+
+```sh
+coder provisioner keys create data-cluster-key --org data-platform
+Successfully created provisioner key data-cluster! Save this authentication token, it will not be shown again.
+
+< key omitted >
+```
+
+Next, on your desired platform, start the provisioner with the key. See our
+[provisioner documentation](../admin/provisioners.md) for details on running on
+additional platforms (e.g. Kubernetes). In this example, we'll start it directly
```

Comment:
Proposing a minor refactor here:

"Next, start the provisioner with the key on your desired platform. In this example, we'll start it using the Coder CLI on a host with Docker. For instructions on using other platforms like Kubernetes, see our
[provisioner documentation](../admin/provisioners.md)."


Alternatively, last line could come after the code snippet.